### PR TITLE
Add clauses about archivebot

### PR DESCRIPTION
**Pull request by kennethkalmer**
Based on the work done in zadev/code-of-conduct#8 with links updated and team renamed where applicable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +86,44 @@ Participants asked to stop any harassing behavior are expected to comply immedia
 
 If a participant engages in harassing behavior, the administrators may take any action they deem appropriate, up to and including expulsion from all ZA Tech spaces and identification of the participant as a harasser to other ZA Tech members and/or the general public.
 
+## Public archiving
+
+Channels may be publicly archived at the discretion of the channel owner or the team admins. This serves three purposes:
+ 
+1. To get around the 10,000 message limit in free Slack teams
```
Comment (imsickofmaps) [inline README.md]: This doesn't seem like a valid reason given we're on a NPO plan now

File: README.md [hunk 2]
```diff
@@ -81,7 +86,44 @@ Participants asked to stop any harassing behavior are expected to comply immedia
 
 If a participant engages in harassing behavior, the administrators may take any action they deem appropriate, up to and including expulsion from all ZA Tech spaces and identification of the participant as a harasser to other ZA Tech members and/or the general public.
 
+## Public archiving
+
+Channels may be publicly archived at the discretion of the channel owner or the team admins. This serves three purposes:
```
Comment (imsickofmaps) [inline README.md]: I would prefer it was an admin only function to ensure we broadcast the change in status to the channel and added a prefix to the topic of the channel to make it very clear to participants

File: README.md (hunk 1 above)
Comment (kennethkalmer) [inline README.md]: Damn, forgot about that clause! Will remove

File: README.md (hunk 2 above)
Comment (kennethkalmer) [inline README.md]: AFAICT there is no way to do this since bots are treated the same as users for the most part

**Pull request closed without merging**